src/java.desktop/share/classes/javax/swing/text/MaskFormatter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2017, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2017, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2000, 2021, Oracle and/or its affiliates. All rights reserved.
```
You just update the second year.